bucket/openttd.json

```
@@ -6,30 +6,30 @@
             "url": [
-                "https://cdn.openttd.org/openttd-releases/1.10.0/openttd-1.10.0-windows-win32.zip",
-                "https://cdn.openttd.org/opengfx-releases/0.6.0/opengfx-0.6.0-all.zip",
-                "https://cdn.openttd.org/opensfx-releases/0.2.3/opensfx-0.2.3-all.zip",
-                "https://cdn.openttd.org/openmsx-releases/0.3.1/openmsx-0.3.1-all.zip"
+                "https://cdn.openttd.org/openttd-releases/12.2/openttd-12.2-windows-win32.zip",
+                "https://cdn.openttd.org/opengfx-releases/7.1/opengfx-7.1-all.zip",
+                "https://cdn.openttd.org/opensfx-releases/1.0.3/opensfx-1.0.3-all.zip",
+                "https://cdn.openttd.org/openmsx-releases/0.4.2/openmsx-0.4.2-all.zip"
             ],
             "hash": [
-                "4f10643582e0c57afeb8b327364d4ff20f3c28e10399d8be7ddf83104b3806cf",
-                "d419c0f5f22131de15f66ebefde464df3b34eb10e0645fe218c59cbc26c20774",
-                "6831b651b3dc8b494026f7277989a1d757961b67c17b75d3c2e097451f75af02",
-                "92e293ae89f13ad679f43185e83fb81fb8cad47fe63f4af3d3d9f955130460f5"
+                "67cd412950080803be3e016f41ae6f369bfc4947f4e30fe694a36f05f43db922",
+                "928fcf34efd0719a3560cbab6821d71ce686b6315e8825360fba87a7a94d7846",
+                "e0a218b7dd9438e701503b0f84c25a97c1c11b7c2f025323fb19d6db16ef3759",
+                "5a4277a2e62d87f2952ea5020dc20fb2f6ffafdccf9913fbf35ad45ee30ec762"
             ]
         },
         "64bit": {
             "url": [
-                "https://cdn.openttd.org/openttd-releases/1.10.0/openttd-1.10.0-windows-win64.zip",
-                "https://cdn.openttd.org/opengfx-releases/0.6.0/opengfx-0.6.0-all.zip",
-                "https://cdn.openttd.org/opensfx-releases/0.2.3/opensfx-0.2.3-all.zip",
-                "https://cdn.openttd.org/openmsx-releases/0.3.1/openmsx-0.3.1-all.zip"
+                "https://cdn.openttd.org/openttd-releases/12.2/openttd-12.2-windows-win64.zip",
+                "https://cdn.openttd.org/opengfx-releases/7.1/opengfx-7.1-all.zip",
+                "https://cdn.openttd.org/opensfx-releases/1.0.3/opensfx-1.0.3-all.zip",
+                "https://cdn.openttd.org/openmsx-releases/0.4.2/openmsx-0.4.2-all.zip"
             ],
             "hash": [
-                "263f0b51d9630a1ef60538e822de064138ea3599f1ace4cc070943e81aa5bc4d",
-                "d419c0f5f22131de15f66ebefde464df3b34eb10e0645fe218c59cbc26c20774",
-                "6831b651b3dc8b494026f7277989a1d757961b67c17b75d3c2e097451f75af02",
-                "92e293ae89f13ad679f43185e83fb81fb8cad47fe63f4af3d3d9f955130460f5"
+                "73f7b6b23dc0c6242c9dbc219128fd49093384f6f09900be7dff00e15723454f",
+                "928fcf34efd0719a3560cbab6821d71ce686b6315e8825360fba87a7a94d7846",
+                "e0a218b7dd9438e701503b0f84c25a97c1c11b7c2f025323fb19d6db16ef3759",
+                "5a4277a2e62d87f2952ea5020dc20fb2f6ffafdccf9913fbf35ad45ee30ec762"
             ]
```

```suggestion
            ],
            "extract_dir": "openttd-12.2-windows-win64",
```

`extract_dir` is needed here. Otherwise installation will fail.

This `extract_dir` will only apply to `openttd-12.2-windows-win64.zip`, because it is actually resolved as:
```
"extract_dir": [
    ""openttd-12.2-windows-win64"
]
```